test/Test_Observer.dart

```diff
@@ -1,103 +1,93 @@
 part of puremvc_unit_tests;
 
-class Test_Observer
-{
-  _tests()
-  {
-    group('Observer', ()
-    {
+class Test_Observer {
+  _tests() {
+    group('Observer', () {
       test('Constructor null args', () {
         // Create a blind Observer
-        mvc.IObserver observer = new mvc.Observer( null, null );
+        mvc.IObserver observer = mvc.Observer(() {}, null);
 
         // Make sure Observer was created
-        expect( observer, isNotNull );
+        expect(observer, isNotNull);
       });
 
       test('Constructor complete args', () {
         // Create an observer with callback method and caller instance defined
-        mvc.IObserver observer = new mvc.Observer( this.observerTestMethod, this );
+        mvc.IObserver observer = mvc.Observer(this.observerTestMethod, this);
 
         // Make sure the Observer was created
-        expect( observer, isNotNull  );
+        expect(observer, isNotNull);
 
         // Make sure caller and callback were set
-        expect( observer.getNotifyMethod(), isNotNull  );
-        expect( observer.notifyMethod, isNotNull  );
-        expect( observer.getNotifyContext(), isNotNull  );
-        expect( observer.notifyContext, isNotNull  );
```

Comment:
why was this test removed?

Reply:
In my view all these access modifiers add confusion to their use cases, add questions - why they needed, how to use them ... looks excess.

Reply:
However I returned them